docs-website/_includes/contribute.html

```diff
@@ -3,7 +3,7 @@
   <em>Last update: {{ page.date }}</em> {% endif %}See something wrong or
   incomplete?
   <a
-    href="https://github.com/BohemianCoding/SketchAPI/tree/develop/docs/{% if page.relative_path %}{{
+    href="https://github.com/sketch-hq/SketchAPI/tree/develop/docs/{% if page.relative_path %}{{
```

Comment:
for further future proofing, maybe we can use `{{ site.github_repo }}` instead of `https://github.com/sketch-hq/SketchAPI` (here and in other pages)